From 17363991d50b21dd06e46e17e73761001838754f Mon Sep 17 00:00:00 2001
From: Florian Schmaus <flo@geekplace.eu>
Date: Wed, 27 Oct 2021 14:32:06 +0200
Subject: [PATCH] Remove broken test

---
 test.py | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/test.py b/test.py
index 63e7b715f517..510b2c005489 100755
--- a/test.py
+++ b/test.py
@@ -497,16 +497,6 @@ class PdfWriterTests(unittest.TestCase):
         self.pdfxml = copy.deepcopy(self.elements_pdfxml)
         self.root = copy.deepcopy(self.elements_root)
 
-    def test_text_content(self):
-        def norm(t):
-            return re.sub(r'\s+', ' ', t).strip()
-        #
-        text = norm('\n'.join( p.text for p in self.pdfxml.xpath('.//Page/text') ))
-        for e in self.root.xpath('./middle//*'):
-            if e.text and e.text.strip() and e.tag not in xml2rfc.util.unicode.unicode_content_tags:
-                t =  norm(e.text.split(None, 1)[0])
-                self.assertIn(t, text)
-
     def test_included_fonts(self):
         if xml2rfc.HAVE_WEASYPRINT and xml2rfc.HAVE_PYCAIRO and xml2rfc.HAVE_CAIRO and xml2rfc.HAVE_PANGO:
             font_families = set([ f.text for f in self.pdfxml.xpath('.//FontFamily') ])
-- 
2.32.0

