From 14df53a2ee468ef767e32ecf148eb991abf9684b Mon Sep 17 00:00:00 2001
From: Florian Schmaus <flo@geekplace.eu>
Date: Mon, 7 Aug 2023 13:44:07 +0200
Subject: [PATCH] setup.cfg: disable pytest coverage

---
 setup.cfg | 2 --
 1 file changed, 2 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index 0c13af35324f..9c08c7f3c17e 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -17,8 +17,6 @@ markers =
 filterwarnings =
     ignore::DeprecationWarning
     default::DeprecationWarning:autosuspend
-addopts =
-    --cov-config=setup.cfg
 
 [coverage:run]
 branch = True
-- 
2.41.0

